util/election2/testonly/tests.go

```diff
@@ -210,7 +210,9 @@ func runElectionClose(t *testing.T, f election2.Factory) {
 		{desc: "master-error", beMaster: true, err: closeErr, wantErr: closeErr},
 		{desc: "not-master", beMaster: false, wantErr: nil},
 		{desc: "not-master-error", err: closeErr, wantErr: closeErr},
+		/* TODO(pavelkalinnikov): reinstate when not flaky.
 		{desc: "cancel", cancel: true, wantErr: nil},
```

Comment:
Hm, if I remember correctly, gofmt usually adds indentations for multi-line comments. Could you run it?

Alternatively:
```suggestion
		// {desc: "cancel", cancel: true, wantErr: nil},
```

Reply:
`gofmt` didn't make any changes, but switched to `//` style comments anyway.